### ISSUE TITLE
Implement Error Overlay

### ISSUE THREAD

**Issue by devloco** [devloco/react-scripts-wptheme#1]
When developing on a laptop you don't always have screen space to keep the watcher window in view. So if you cause an error, your browser simply doesn't refresh and you have to switch over to the watcher window to see what's up.

The original create-react-app has a nice error overlay that appears in the browser when there is an error. Unfortunately their error overlay is tied to their implementation of Hot Module Replacement, which in turn is tied to the Webpack Dev Server. And thus we're at an impasse as the whole point of create-react-wptheme is to get rid of Webpack Dev Server so that we can work within our WordPress servers.

Basically, I'm going to have to write my own Error Overlay. I think I can import the create-react-app tools and make the overlay the same as theirs, but it'll need to be triggered via the wpThemeServer (which currently does nothing more than tell the wpThemeClient to refresh the browser after a successful compile by Webpack).

----
From the To Do list on create-react-wptheme:

see: TODO note in wpstart.js.
see error overlay stuff here:
https://github.com/facebook/create-react-app/blob/master/packages/react-dev-utils/webpackHotDevClient.js

**Comment by devloco** [comment]
I finished this a while back, and forgot to close this issue.
create-react-wptheme now uses this for error overlay:
https://github.com/devloco/react-scripts-wptheme-error-overlay
